matcher/service/openreview_interface.py

```diff
@@ -815,23 +815,28 @@ def run(self):
         try:
             venue = None
             self.config_note_interface.set_status(MatcherStatus.DEPLOYING)
+            support_user = 'openreview.net/Support'
```

Comment:
is this always going to be `openreview.net/Support`?

Reply:
thanks for the catch, I forgot to revert the change.